app/services/flood_risk_engine/find_address_by_uprn.rb

```diff
@@ -0,0 +1,30 @@
+module FloodRiskEngine
+  class FindAddressByUprn
+
+    attr_reader :address_data
+
+    def initialize(uprn)
+      Rails.logger.debug("FindAddressByuPRN supplied with params [#{uprn.inspect}]")
+      @uprn = uprn
+    end
+
+    def search
+      @address_data = nil
+
+      @address_data = begin
+        EA::AddressLookup.find_by_uprn(uprn)
+      rescue => ex
+        Airbrake.notify(ex) if defined? Airbrake
+        Rails.logger.error "Address lookup service failed: #{ex}"
+        nil
+      end
+
```

Comment:
I might be wrong (sorry if so) but I don't think `@address_data` or the `address_data` are used anywhere so search could be
```
def search
  EA::AddressLookup.find_by_uprn(uprn)
rescue => ex
  Airbrake.notify(ex) if defined? Airbrake
  Rails.logger.error "Address lookup service failed: #{ex}"
  nil
end
```